src/registrar/fixtures.py

```diff
@@ -73,7 +99,58 @@ def load(cls):
                 logger.debug("User object created for %s" % admin["first_name"])
             except Exception as e:
                 logger.warning(e)
-        logger.debug("All users loaded.")
+        logger.debug("All superusers loaded.")
+
+        logger.info("Going to load %s CISA analysts (staff)" % str(len(cls.STAFF)))
+        for staff in cls.STAFF:
+            try:
+                user, _ = User.objects.get_or_create(
+                    username=staff["username"],
+                )
+                user.is_superuser = False
+                user.first_name = staff["first_name"]
+                user.last_name = staff["last_name"]
+                user.is_staff = True
+                user.is_active = True
+
+                for permission in cls.STAFF_PERMISSIONS:
+                    app_label = permission["app_label"]
+                    model_name = permission["model"]
+                    permissions = permission["permissions"]
+
+                    # Retrieve the content type for the app and model
+                    content_type = ContentType.objects.get(
+                        app_label=app_label, model=model_name
+                    )
+
+                    # Retrieve the permissions based on their codenames
+                    permissions = Permission.objects.filter(
+                        content_type=content_type, codename__in=permissions
+                    )
+
+                    # Assign the permissions to the user
+                    user.user_permissions.add(*permissions)
+
+                    # Convert the permissions QuerySet to a list of codenames
+                    permission_list = list(
+                        permissions.values_list("codename", flat=True)
+                    )
+
+                    logger.debug(
+                        app_label
+                        + " | "
+                        + model_name
+                        + " | "
+                        + ", ".join(permission_list)
+                        + " added for user "
+                        + staff["first_name"]
+                    )
+
+                user.save()
+                logger.debug("User object created for %s" % staff["first_name"])
+            except Exception as e:
+                logger.warning(e)
+        logger.debug("All CISA analysts (staff) loaded.")
```

Comment:
same as above, I would make this info, just given the log that stays it is going to load is also info.